docs/ldml/tr35-dates.md

```diff
@@ -2697,13 +2705,23 @@ This means that the Year, Month, and Day semantic field mapping in en-US should
 | Month          | Japanese  | MMMM   | MMM    | M      |
 | Day            | Japanese  | d      | d      | d      |
 
+##### <a name="Semantic_Time_Precision_Skeleton_Variations" href="#Semantic_Time_Precision_Skeleton_Variations">Time Precision Skeleton Variations</a>
+
+The [time precision](#Semantic_Skeleton_Time_Precision) should change the skeleton for all lengths as follows:
+
+- Hour: No change.
+- Minute: Add "m"
```

Comment:
Add "mm"? Same below, you're adding the padded minutes/seconds, right?

Reply:
availableFormats will fix the length if necessary for the locale.